hydra-node/src/Hydra/Network/Ouroboros.hs

```diff
@@ -48,6 +48,7 @@ import Hydra.Network.Ouroboros.Type (
   Message (..),
   codecFireForget,
  )
+import Hydra.Prelude
```

Comment:
Seems like this import was moved accidentally here.

Should: apply our code style of prelude imports as a separate block above